script_mteb_french/run_benchmark.py

```diff
@@ -135,15 +134,7 @@
     "FloresBitextMining",
 ]
 
-TASKS = (
-    TASK_LIST_CLASSIFICATION
-    + TASK_LIST_CLUSTERING
-    + TASK_LIST_PAIR_CLASSIFICATION
-    + TASK_LIST_RERANKING
-    + TASK_LIST_RETRIEVAL
-    + TASK_LIST_STS
-    + TASK_LIST_SUMMARIZATION
-)
+TASKS = TASK_LIST_BITEXTMINING
```

Comment:
Sure about merging the modifications in this script ?

Reply:
I juste rebase maininto this branch, everything should be fixed now :)